inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -378,18 +378,23 @@ int main(int argc, char *argv[]) {
             topology_name = cnnNetwork.getName();
             slog::info << (FLAGS_b != 0 ? "Network batch size was changed to: " : "Network batch size: ") << batchSize << slog::endl;
 
-            // ----------------- 6. Configuring input ----------------------------------------------------------------------
+            // ----------------- 6. Configuring inputs and outputs ----------------------------------------------------------------------
             next_step();
 
-            for (auto& item : inputInfo) {
+            processPrecision(cnnNetwork, FLAGS_ip, FLAGS_op, FLAGS_iop);
+            for (auto& item : cnnNetwork.getInputsInfo()) {
+                /** Set the precision of input data provided by the user, should be called before load of the network to the device **/
                 if (app_inputs_info.at(item.first).isImage()) {
```

Comment:
If input is not image, but precision was set via command line we need update information too

Reply:
Fixed.